test/adaptive_load/adaptive_load_client_main_test.cc

```diff
@@ -4,6 +4,7 @@
 #include "nighthawk/adaptive_load/adaptive_load_controller.h"
 #include "nighthawk/common/exception.h"
 
+#include "external/envoy/src/source/common/protobuf/protobuf.h"
```

Comment:
This seems to be causing a test failure, looks like we may need to add this as a dep in the BUILD file too.

However, please see mu other comment first.

Reply:
I got a wrong path. Now it should be fixed.